docs/End_to_End_Testing.md

```diff
@@ -0,0 +1,65 @@
+# End-to-End Testing
+
+End-to-end test must be included as a part of a PR for any contribution to Flux including: 
+[Flux Functions](/Flux_Functions.md), 
+[Standalone Scalar Functions](/Scalar_Functions.md), 
+[Sink/Source Functions](/Source_Sink_Functions.md), 
+[Stream Transformation Functions](/Stream_Transformation_Functions.md). 
+
+## Required guidelines
+
+Please help us make the contribution process easier by providing feedback about your experience and any technical hurdles you encountered here.
+
+### Pure Flux Code Functions Guidelines
+
+If you add a new [test](../stdlib/testing/testdata), our test framework automatically detects the file and tests it. To ensure your file passes the test, consider the correct form of simple_max.flux:

```

Comment:
This directory doesn't exist anymore, tests are now adjacent to their go files.

For example see `stdlib/univers/columns_test.flux`

You should specify the guidelines for writing a test files:

 - the file name must end with `_test.flux`
 - the package must be named after the containing folder. For example, in `universe`, the test must start with   `package universe_test`.

Also,  remember to  `make generate` before adding files to the commit.

Reply:
I think you solved this in the last commit 👍